### ISSUE TITLE
Multi-label classification `predict` method return probabilities rather than calibrated labels.

### ISSUE THREAD

**Issue by qingquansong** [keras-team/autokeras#1122]
### Bug Description

Multi-label classification `predict()` method return probabilities rather than calibrated labels.

### Bug Reproduction
https://github.com/datamllab/automl-in-action-notebooks/blob/master/3.3.4-Task-API-Multi-label.ipynb

### Setup Details
Include the details about the versions of:
 - OS type and version:
 - Python: 
 - autokeras: 1.0.2
 - keras-tuner:
 - scikit-learn:
 - numpy:
 - pandas:
 - tensorflow: 2.1.0
